### DISCUSSION TITLE
Repo Install on Rocky 9

### DISCUSSION THREAD

**Discussion by Voroxpete** [45Drives/cockpit-file-sharing#71] [General]
I see that repo install is supported for Rocky linux 8.  Is there any reason why Rocky 9 isn't also supported?  Just wondering.

**Comment by malcie** [comment]
Rocky9 is working like a charm. 

sudo dnf config-manager --set-enabled crb

sudo dnf -y install \
    https://dl.fedoraproject.org/pub/epel/epel-release-latest-9.noarch.rpm \
    https://dl.fedoraproject.org/pub/epel/epel-next-release-latest-9.noarch.rpm

dnf -y install cockpit-file-sharing cockpit

systemctl enable cockpit --now
